mindjunkies/dashboard/tests/test_views.py

```
@@ -1,11 +1,12 @@
 import pytest
-from django.urls import reverse
-from model_bakery import baker
 from django.contrib.messages import get_messages
 from django.http import HttpResponseRedirect
+from django.urls import reverse
+from model_bakery import baker
+
 from mindjunkies.accounts.models import User
```

Ruff (0.8.2)

`mindjunkies.accounts.models.User` imported but unused

Remove unused import: `mindjunkies.accounts.models.User`

(F401)